README.md

```diff
@@ -69,7 +69,7 @@ optional. Default: `⚠️ Nightly upstream-dev CI failed ⚠️`
 In case you don't like the default title for new issues, this setting can be used to set a different one:
 
 ```yaml
-- uses: xarray-contrib/issue-from-pytest-log@v1
+- uses: scientific-python/issue-from-pytest-log-action@main
```

Comment:
I don't think it's correct to recommend using the `main` for any actions, let alone one that opens issues. 

Can we change back the versions to preferably recommend a hash, or at worst a version alias?

(hash: while this is not a release action and thus it's not critical; for consistency the recommendation is to pin to a hash: https://github.com/scientific-python/specs/issues/392#issue-3064492281. Dependabot will deal with it just fine.